restconnector/RequestHandler.cpp

```diff
@@ -25,9 +27,13 @@ RequestHandler::RequestHandler( const std::string& publisherSchema,
     , publisher_( new ::zeq::Publisher( lunchbox::URI( publisherSchema ) ) )
     , listening_( true )
     , listeningThread_( new boost::thread( boost::bind( &RequestHandler::listen_, this ) ) )
+    , renderingRessourceStrarted_( false )
 {
-    subscriber_->registerHandler( ::zeq::hbp::EVENT_IMAGEJPEG,
-                                  boost::bind( &RequestHandler::onImageJPEGEvent_, this, _1 ));
+    ZeqEventDescriptor eventDescriptor;
+    eventDescriptor.zeqEvent_ = ::zeq::hbp::EVENT_IMAGEJPEG;
+    eventDescriptor.schema_ = ::zeq::hbp::SCHEMA_IMAGEJPEG;
```

Comment:
You can have an "empty" ctor, and one with two parameters. STL is happy and you can still map.insert( ZeqEventDescriptor( val, val ));